components/widgets/inputs/users_emails_input.jsx

```diff
@@ -323,6 +366,7 @@ export default class UsersEmailsInput extends React.PureComponent {
                                 </components.NoOptionsMessage>
                             )}
                         </FormattedMarkdownMessage>
+                        <NotifyButton/>
```

Comment:
What is the condition to show this new `NotifyButton`? Given that this is a shared component we don't want to show this for all but only when end-users have reached the limit of invites in cloud. 

I believe you can leverage on @pablovelezvidal work to add the `Upgrade Now` link that was added in a generic way to this component to avoid adding cloud specific logic here: https://github.com/mattermost/mattermost-webapp/pull/7446

Reply:
@marianunez is right, I already added the same logic and instead of hardcoding the element here, I send it as a new prop, so it is customizable from the component that calls the user_emails_input.

Reply:
@marianunez and @pablovelezvidal I think it's best if I wait for #7446 to merge and then build on top of it, since Pablo already has a good way to add these links (Upgrade and Notify the Admin) in that PR

Reply:
@AGMETEOR Pablos PR has been merged and should unblock you here

Reply:
@marianunez and @pablovelezvidal closing in favor of #7546